release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3318,7 +3318,50 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
-// 4.14.23
+[id="ocp-4-14-24"]
+=== RHSA-2024:2668 - {product-title} 4.14.24 bug fix update and security update
+
+Issued: 2024-05-09
+
+{product-title} release 4.14.24, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:2668[RHSA-2024:2668] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:2672[RHSA-2024:2672] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.24 --pullspecs
+----
+
+[id="ocp-4-14-24-enhancements"]
+==== Enhancements
+
+The following enhancements are included in this z-stream release:
+
+[id="ocp-4-14-ipv6-default-macvlan"]
+==== IPv6 unsolicited neighbor advertisements now default on macvlan CNI plug-in
+
+Pods created using the macvlan CNI plug-in, where the IP address management CNI plug-in has assigned IP addresses, now send IPv6 unsolicited neighbor advertisements by default onto the network. This notifies hosts of the new pod's MAC address for a particular IP address to refresh IPv6 neighbor caches. (link:https://issues.redhat.com/browse/OCPBUGS-33066[*OCPBUGS-33066*])
+
+[id="ocp-4-14-24-bug-fixes"]
+==== Bug fixes
+
+* Previously, when a cluster was installed using proxy and the proxy information contained escape characters in the format "%XX" the installation would fail. With this release, the issue has been fixed. (link:https://issues.redhat.com/browse/OCPBUGS-33010[*OCPBUGS-33010*])
```

Comment:
```suggestion
* Previously, when a cluster was installed using proxy and the proxy information contained escape characters in the format `%XX` the installation would fail. With this release, the issue has been fixed. (link:https://issues.redhat.com/browse/OCPBUGS-33010[*OCPBUGS-33010*])
```